rpm/include.am

```diff
@@ -4,6 +4,12 @@ rpm-build: rpm/spec dist
 	@rm -f *.rpm
 	rpmdev-setuptree
 	find ~/rpmbuild/RPMS ~/rpmbuild/SRPMS -name "$(PACKAGE)-$(VERSION)*.rpm" | xargs rm -f
+	@rm -Rf $(PACKAGE)-$(VERSION)-stable
+	@tar xzf $(PACKAGE)-$(VERSION).tar.gz
+	@mv $(PACKAGE)-$(VERSION) $(PACKAGE)-$(VERSION)-stable
+	@cp autogen.sh $(PACKAGE)-$(VERSION)-stable
```

Comment:
autogen.sh is really for recreating the configure file and initial setup after the repository is cloned. Existing bundles, or new bundles created with make dist, should not need autogen.sh though. If running into a case where configure.ac needs altered and configure needs re-created `autoreconf -fi` could be ran.